posts/seurat-cosmx-basics/index.qmd

```diff
@@ -0,0 +1,244 @@
+---
+title: "Introduction to visualizing CosMx data in Seurat"
+author:
+  - name: Claire Williams
+    affiliations:
+      - ref: nstg
+      - ref: crwilliams11
+affiliations:
+  - id: nstg
+    name: NanoString Technologies, Inc.
+    city: Seattle
+    state: WA
+  - id: crwilliams11
+    name: "Github: [@crwilliams11](https://github.com/crwilliams11)"
+toc: true
+toc-title: Contents
+toc-depth: 3
+toc-expand: 2
+toc-location: left
+number-sections: true
+number-depth: 4
+date: "2024-05-10"
+date-modified: last-modified
+categories: [visualization]
+draft: false
+---
+
+# Introduction 
+
+This is a non-comprehensive vignette to introduce the use of the Seurat package to visualize and work with data generated from the CosMx&#8482; Spatial Molecular Imager (SMI). This vignette does not cover analysis of data in Seurat but rather tries to address frequently asked questions we've received from customers on getting started with their data in Seurat. As additional resources, please see vignettes on the Seurat website: [Spatial Vignette](https://satijalab.org/seurat/articles/seurat5_spatial_vignette_2.html#human-lung-nanostring-cosmx-spatial-molecular-imager) and
```

Comment:
I think this is a good first pass but I would like to see some rephrasing of the introduction to set the stage for the reader. Here's my attempt: 

One of the most exciting aspects of CosMx data is the ability to directly observe gene expression with spatial context. This is a great technological leap from the spatially disruptive approaches. When doing spatial data analysis, the Seurat R package (link) has continually added features to support CosMx data. And readers are encouraged to take a look at previous vignettes by the Seurat group (links here) as well as works we've provided previously (links to scratch space). The blog post herein supplements these and provides you with some of the configurations that we find helpful as you explore your CosMx data.

Reply:
Good idea. I merged what I had with this suggestion to come to the following, in commit 54f276a:
"One of the most exciting aspects of CosMx™ Spatial Molecular Imager (SMI) data is the ability to directly observe gene expression in its spatial context at the single cell level. This is a great technological leap from previous single cell transcriptomics methods that lost spatial context while retrieving cells. For analysts looking to perform spatial data analysis, the [Seurat R package](https://satijalab.org/seurat/) has continually added features to support CosMx data. Readers are encouraged to take a look at previous vignettes by the Seurat group ([Spatial Vignette](https://satijalab.org/seurat/articles/seurat5_spatial_vignette_2.html#human-lung-nanostring-cosmx-spatial-molecular-imager) and [Clustering Tutorial](https://satijalab.org/seurat/articles/pbmc3k_tutorial)) as well as blog posts we’ve provided previously ([scratch space](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html)). The blog post herein supplements these and provides you with some of the plotting configurations we find most helpful as you explore your CosMx data. This vignette does not cover analysis of data in Seurat but rather tries to address frequently asked questions we’ve received from customers on getting started with their data in Seurat."